### PR TITLE
Fixes ratfish harddels, minor code improvements to moisture traps

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Fix:
Fixes the ratfish spawning by removing instafish and making moisture traps properly generate the item in themselves.
Code Improvement:
dmdoc format for loot table
loot table var given a bit more precise of a name
loot table is now static between all moisture traps
when you get bitten, the bite message comes before the damage.

## Why It's Good For The Game

Master is BROKEN right now!

## Changelog

:cl:
fix: Ratfish can now properly show up in moisture traps
code: Moisture traps are coded a little better now
spellcheck: fixed some wording in the moisture trap bite feedback
/:cl:

### CODE REVIEW

File: code/game/objects/structures/maintenance.dm [hunk 1]
```diff
@@ -74,9 +78,8 @@ at the cost of risking a vicious bite.**/
 	if(critter_infested && prob(50) && iscarbon(user))
 		var/mob/living/carbon/bite_victim = user
 		var/obj/item/bodypart/affecting = bite_victim.get_bodypart("[(user.active_hand_index % 2 == 0) ? "r" : "l" ]_arm")
+		to_chat(user, span_danger("You feel a sharp as an unseen creature sinks it's [pick("fangs", "beak", "proboscis")] into your arm!"))
```
Comment (SuperNovaa41) [inline code/game/objects/structures/maintenance.dm:81]: You feel a sharp?

Comment (tralezab) [reply code/game/objects/structures/maintenance.dm:81]: Wow... the old message said that? How did I not notice, hahaha

**Comment by Mothblocks** [comment]
@LemonInTheDark Why does your test only work half the time

**Comment by LemonInTheDark** [comment]
> @LemonInTheDark Why does your test only work half the time

Fool you fell for my trap

Also you shoulda just made moisture traps use contents here, woulda been a more general fix, and fit their usecase better

**Pull request merged**